src/components/PageHeader.js

```diff
@@ -6,7 +6,7 @@ function PageHeader({ username, avatarUrl }) {
     <Navbar>
       <Navbar.Header>
         <Navbar.Brand>
-          <a href="/">source{'{d}'} code annotation</a>
+          <h1><a href="/"><img alt="source{d} code annotation" src="/sourced-code-annotation-logo.svg" />code annotation</a></h1>
```

Comment:
The CI fails because this piece of code is not properly indented.
It should be:
```jsx
        <Navbar.Brand>
          <h1>
            <a href="/">
              <img
                alt="source{d} code annotation"
                src="/sourced-code-annotation-logo.svg"
              />
              code annotation
            </a>
          </h1>
        </Navbar.Brand>
```

Reply:
@dpordomingo Thanks, done!